lib/updateVersion.js

```diff
@@ -804,7 +804,7 @@ function moveByteStrideToBufferView(gltf) {
                         accessor.bufferView = newBufferViewId;
                         accessor.byteOffset = accessor.byteOffset - currentByteOffset;
                     }
-                    currentByteOffset = accessorByteOffset + accessorByteLength;
+                    currentByteOffset = accessorByteOffset + accessorByteLength + accessorByteLength % 4;
```

Comment:
@ggetz I stared at this for a really long time. The fix here is pretty close, but I think it needs to be

`currentByteOffset = (i < accessorsLength - 1) ? accessors[i + 1].byteOffset : undefined;`

Which is setting the current byte offset to next accessor's byte offset rather than the end of the current accessor.
Cesium_Air brought up a discrepancy where these two values are not always the same if there is padding between the current accessor and the next accessor.

This approach should work in Cesium without requiring the more involved changes of always aligning byte offsets and byte strides to 4 bytes, which I still think involves buffer copies of some sort.

Reply:
Got it, thanks @lilleyse! This means we don't have to pad the buffer correct? That's good news!

Reply:
Yeah no padding required for this fix thankfully.